src/components/quizCard/QuizCard.jsx

```diff
@@ -0,0 +1,33 @@
+import "./css/quizCardStyle.css";
+
+export default function QuizCard(props) {
```

Comment:
instead of writing props, destructure it. for e.g. { quizDetails }, also you can further destructure quizDetails properties  in component

Reply:
Will take care of that now.